src/core/util/util.js

```diff
@@ -633,6 +633,42 @@ function UtilFactory($document, $timeout, $compile, $rootScope, $$mdAnimate, $in
         if (!template || !angular.isString(template)) return template;
         return template.replace(/\{\{/g, startSymbol).replace(/}}/g, endSymbol);
       }
+    },
+
+    /**
+     * Scan up dom hierarchy for enabled parent;
+     */
+    getParentWithPointerEvents: function (element) {
+      var parent = element.parent();
+
+      // jqLite might return a non-null, but still empty, parent; so check for parent and length
+      while (this.hasComputedStyleValue('pointer-events','none', parent[0] || element[0])) {
```

Comment:
Do no use `this.` here. The `$mdUtil.getParentWithPointerEvents( )` is a generic util that cannot require the `this` scope to be set.